runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineDebugOptions.java

```diff
@@ -191,6 +191,18 @@ public Dataflow create(PipelineOptions options) {
   void setOverrideWindmillBinary(String value);
 
   /**
+   * Custom windmill service endpoint.
+   */
+  @Description("Custom windmill service endpoint.")
```

Comment:
Essentially localhost is the default when this is not set. The clients for local windmill server (the one in prod) and to remote service take different code paths (one uses JNI and c++ client, other uses grpc for google cloud).